sale_double_validation/models/sale.py

```diff
@@ -1,12 +1,14 @@
 # Copyright 2017 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
-from odoo import api, models, _
+from odoo import api, models, fields, _
 from odoo.tools import float_compare
 
 
 class SaleOrder(models.Model):
     _inherit = 'sale.order'
 
+    state = fields.Selection(default='to_approve')
```

Comment:
Wouldn't this set the default state to `to_approve` for all the quotations/sale orders? Even for those that `is_to_approve() == False`?

Reply:
yes but if is_to_approve() == False it will be moved to draft on the create

Reply:
IMHO, changing this default behaviour could lead to errors (e.g.: you can see in tests). This change should be set by option.